lib/auth/grpcserver.go

```diff
@@ -3623,17 +3623,43 @@ func (g *GRPCServer) CreateSessionTracker(ctx context.Context, req *proto.Create
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
-	session, err := auth.ServerWithRoles.CreateSessionTracker(ctx, req)
+
+	var createTracker types.SessionTracker = req.SessionTracker
+	// DELETE IN 11.0.0
+	// Early v9 versions use a flattened out types.SessionTrackerV1
+	if req.SessionTracker == nil {
```

Comment:
Looks like you have identical logic in the API client. Maybe turn this into a method, which will be easier to deprecate.

Reply:
Actually they are reciprocals, req -> sessionTracker and sessionTracer -> req.